diff --git a/node_modules/kbar/lib/KBarAnimator.d.ts b/node_modules/kbar/lib/KBarAnimator.d.ts
index f87a9b4..934c3b1 100644
--- a/node_modules/kbar/lib/KBarAnimator.d.ts
+++ b/node_modules/kbar/lib/KBarAnimator.d.ts
@@ -3,5 +3,5 @@ interface KBarAnimatorProps {
     style?: React.CSSProperties;
     className?: string;
 }
-export declare const KBarAnimator: React.FC<KBarAnimatorProps>;
+export declare const KBarAnimator: React.FC<React.PropsWithChildren<KBarAnimatorProps>>;
 export {};
diff --git a/node_modules/kbar/lib/KBarContextProvider.d.ts b/node_modules/kbar/lib/KBarContextProvider.d.ts
index 62ebd76..10ca955 100644
--- a/node_modules/kbar/lib/KBarContextProvider.d.ts
+++ b/node_modules/kbar/lib/KBarContextProvider.d.ts
@@ -1,4 +1,4 @@
 import * as React from "react";
 import type { IKBarContext, KBarProviderProps } from "./types";
 export declare const KBarContext: React.Context<IKBarContext>;
-export declare const KBarProvider: React.FC<KBarProviderProps>;
+export declare const KBarProvider: React.FC<React.PropsWithChildren<KBarProviderProps>>;
diff --git a/node_modules/kbar/lib/KBarPositioner.d.ts b/node_modules/kbar/lib/KBarPositioner.d.ts
index cdb92cb..f4a50e2 100644
--- a/node_modules/kbar/lib/KBarPositioner.d.ts
+++ b/node_modules/kbar/lib/KBarPositioner.d.ts
@@ -3,5 +3,5 @@ interface Props {
     className?: string;
     style?: React.CSSProperties;
 }
-export declare const KBarPositioner: React.FC<Props>;
+export declare const KBarPositioner: React.FC<React.PropsWithChildren<Props>>;
 export {};
